static/sass/_pattern_strip.scss

```diff
@@ -244,3 +245,18 @@
     }
   }
 }
+
+@mixin canonical-p-strip-suru-topped {
+  .p-strip--suru-topped {
+    @extend %vf-strip;
+
+    background-blend-mode: multiply, multiply, normal, normal;
```

Comment:
can you please add a fallback for no multiply support likewe have on [ubuntu](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/sass/_pattern_strip.scss#L71)

Reply:
I copied it from Ubuntu so we'll have to fix it there too

Reply:
I had a look into it and it looks fine without the blend mode set. The design still looks good and doesn't intersect content to there are no transparency issues.